src/Command/UpdateLicensesCommand.php

```diff
@@ -420,4 +422,16 @@ private function printDryRunPrettyReport(InputInterface $input, OutputInterface
         $style->text('Files with bad license headers:');
         $style->listing($report['fixed']);
     }
+
+    /**
+     * Closure to filter parsed files.
+     *
+     * @see https://symfony.com/doc/3.4/components/finder.html#custom-filtering
+     */
+    protected function getFinderFilter()//: \Closure
+    {
+        return function (\SplFileInfo $file) {
+            return !in_array($file->getBasename(), static::DEFAULT_IGNORED_FILES);
```

Comment:
I see what you expect people wanting to customize "what files should be ignored" by overriding the const DEFAULT_IGNORED_FILES using inheritance.

We can make this even easier 🤔 :
1. Use a property instead of a const
```suggestion
            return !in_array($file->getBasename(), $this->filesToIgnore);
```
2. Property can be set as `protected` so children classes can mutate it
3. A `setfilesToIgnore()` function can be added so you dont even need to extend the class, you can do
```php
class FixLicenses extends UpdateLicensesCommand
{
    protected function configure()
    {
    	$this->setfilesToIgnore(...);

        parent::configure();
        $this
            ->setName('fop:fix-licenses');
    }
```

With a public setter, you could allow people 😉 
- to use this `UpdateLicenseCommand` directly `$a = new UpdateLicenseCommand; $a->setfilesToIgnore(...);`
- to define at runtime what files should be ignored (a const must be an immutable value)

Reply:
Good catches !
Thanks for feedback 👍 
I'll consider all this, when I'll be back home, in a week, it's holidays for now.🌳 🚶 🍂 🌰